go/libraries/doltcore/sqle/kvexec/lookup_join.go

```diff
@@ -193,11 +192,20 @@ type lookupMapping struct {
 	srcVd      *val.TupleDesc
 	srcMapping val.OrdinalMapping
 	// litTuple are the statically provided literal expressions in the key expression
-	litTuple val.Tuple
-	split    int
+	litTuple   val.Tuple
+	split      int
+	keyExprs   []sql.Expression
+	idxColTyps []sql.ColumnExpressionType
 }
 
-func newLookupKeyMapping(ctx context.Context, sourceSch schema.Schema, tgtKeyDesc *val.TupleDesc, keyExprs []sql.Expression, typs []sql.ColumnExpressionType, ns tree.NodeStore) (*lookupMapping, error) {
+func newLookupKeyMapping(
+	ctx *sql.Context,
```

Comment:
Honestly glad that this change was made. Eventually, we should have a `*sql.Context` throughout the entire Dolt codebase since that's the direction we've moved in anyway, but that's a far, _far_ different goal from this PR. All of the `context.Context` params are SQL ones anyway.

Reply:
lol yeah